### PR TITLE
Fix/1571/count of applications

### PR DESCRIPTION
**Pull request by aquibmoin786**
#1571

### Description

- I have added new popup icon Total Count on Recruitment page. Clicking Total Count, We have getting RoundWise Application counting.  

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by P4NK4J** [comment]
@aquibmoin786 Please see the screenshot below, its throwing an error. Seems like total-count-icon blade is not available on GitHub. Please check.
image

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left a comment, please check.

File: Modules/HR/Http/Controllers/Recruitment/ApplicationController.php [hunk 1]
```diff
@@ -131,9 +131,12 @@ public function index()
             })
             ->count();
         }
+
         $attr['jobs'] = Job::all();
         $attr['universities'] = University::all();
         $attr['tags'] = Tag::orderBy('name')->get();
+        $attr['rounds'] = $hrRounds;
+        $attr['rounds'] = $hrRounds;
```
Comment (GauravGusain98) [inline Modules/HR/Http/Controllers/Recruitment/ApplicationController.php:139]: I think this is duplicate. Please remove this.

File: resources/views/hr/application/total-count-modal.blade.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+  <!-- Modal -->
+  <div class="modal fade" id="myModal" role="dialog">
```
Comment (GauravGusain98) [inline resources/views/hr/application/total-count-modal.blade.php:2]: Don't use the variable, id or class names as `myModal`. We should use the names that give clarity on what this element or variable is doing. 

So please update the name to a more meaningful name.

File: resources/views/hr/menu.blade.php [hunk 3]
```diff
@@ -10,8 +10,11 @@
     </li>
     <li class="nav-item">
         <a class="nav-item nav-link {{ Request::is('hr/recruitment/reports*') ? 'active' : '' }}" href="{{ route('recruitment.reports') }}"><i class="fa fa-pie-chart"></i>&nbsp;Reports</a>
-    </li>
+    </li> 
     <li class="nav-item">
         <a class="nav-item nav-link {{ Request::is('hr/recruitment/campaigns*') ? 'active' : '' }}" href="{{ route('recruitment.campaigns') }}"><i class="fa fa-envelope"></i>&nbsp;Campaigns</a>
     </li>
+    <li class="nav-item" data-toggle="modal" data-target="#myModal">
```
Comment (GauravGusain98) [inline resources/views/hr/menu.blade.php:17]: Update the name here as well

File: resources/views/hr/menu.blade.php [hunk 4]
```diff
@@ -10,8 +10,11 @@
...
+        <a class="nav-item nav-link"><i class="fa" style="font-size:24px">&#xf07a;</i>&nbsp;Total Count</a>
```
Comment (GauravGusain98) [inline resources/views/hr/menu.blade.php:18]: We don't use style attribute inside the blade. Please use the class name that is provide for increasing the font size

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments

**Comment by aquibmoin786** [comment]
I'm facing some problem in this branch. So I close this branch.

**Pull request closed without merging**